### ISSUE TITLE
Crash with 1.8

### ISSUE THREAD

**Issue by kevinburkeshyp** [equinox-io/equinox#3]
When I type `equinox -h`, I get the following error:

```
panic: Failed to load asset pkg/Distribution: Asset assets/pkg/Distribution can't read by error: Error reading asset assets/pkg/Distribution at /Users/burke/code/go/src/lib/packaging/assets/pkg/Distribution: open /Users/burke/code/go/src/lib/packaging/assets/pkg/Distribution: no such file or directory

goroutine 1 [running]:
lib/packaging.init·2()
    /Users/kyle/projects/equinox/src/lib/packaging/pkg.go:29 +0x1a6
lib/packaging.init()
    /Users/kyle/projects/equinox/src/lib/packaging/zip.go:28 +0x828
lib/worker.init()
    /Users/kyle/projects/equinox/src/lib/worker/queue.go:64 +0x47
lib/tool.init()
    /Users/kyle/projects/equinox/src/lib/tool/update.go:43 +0x9b
main.init()
    /Users/kyle/projects/equinox/src/tool/main.go:11 +0x42

goroutine 8 [runnable]:
github.com/getsentry/raven-go.(*Client).worker(0xc208106000)
    /Users/kyle/projects/equinox/vendor/src/github.com/getsentry/raven-go/client.go:414
created by github.com/getsentry/raven-go.newClient
    /Users/kyle/projects/equinox/vendor/src/github.com/getsentry/raven-go/client.go:304 +0x19a
```

I'm not sure how to fix it. the referenced files don't exist, and googling for the error string didn't turn up anything obvious.

**Comment by kevinburkeshyp** [comment]
Version 1.7.2 works fine

**Comment by inconshreveable** [comment]
Fixed with 1.8.1.
